content/en/acknowledgements.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Acknowledgements
+menu: { main: { weight: 100 } }
+---
+
+{{% blocks/lead color="primary" %}}
+# Acknowledgements
+{{% /blocks/lead %}}
+
+{{% blocks/section type="section" color="white" %}}
```

Comment:
If we move this page to under `/docs`, as was suggested earlier, then we'll need to drop the `blocks/*` shortcodes.